lib/bamboo/adapters/sendgrid_helper.ex

```diff
@@ -0,0 +1,77 @@
+defmodule Bamboo.SendgridHelper do
+  @moduledoc """
+  Functions for using features specific to Sendgrid template substitution tags.
```

Comment:
That is probably a good idea. Are you thinking of just altering the first two sections or also abandon the provided example usage?